pages/common/go-test.md

```diff
@@ -0,0 +1,24 @@
+# go test
+
+> Tests Go packages (files have to end with `_test.go`).
+> More information: <https://golang.org/cmd/go/#hdr-Testing_flags>.
+
+- Test the package found in the current directory:
+
+`go test`
+
+- [v]erbosely test the package in the current directory:
+
+`go test -v`
+
+- Test the packages in the current directory and all subdirectories:
+
+`go test -v ./...`
+
+- Test the package in the current directory and run all benchmarks:
+
+`go test -v -bench .`
+
+- Test the package in the current directory and run all benchmarks for 50 seconds:
+
+`go test -v -bench . -benchtime 50s`
```

Comment:
```suggestion
`go test -v -bench {{.}} -benchtime {{50s}}`
```

Reply:
I suppose this reply relates to the the previous comment/suggestion, but the `.` would actually be interpreted as a regular expression if changed. I never mentioned this in the examples as I assumed this would break the 5th guideline mentioned in the contributing guidelines, that being
> Focus on details specific to the command, and avoid explaining general UNIX concepts that could apply to any command (ex: relative/absolute paths, glob patterns/wildcards, special character escaping...).

So I just attempted to keep the examples simple in that regard.

If that really is the case, per my understanding the curly brackets wouldn't be necessary around the `.` character.

Reply:
You're right, it's good to avoid introducing general UNIX concepts; however, regular expressions is really a general computer science concept, and use on lots and lots of other pages, so I think it's okay (and advisable) to clarify that here. 

>If that really is the case, per my understanding the curly brackets wouldn't be necessary around the `.` character.

We always put the token syntax (`{{` and `}}`) around user-mutable parameters/arguments.  If the `.` should always be a `.` to make the example correct, then let's leave it as-is.  Otherwise, if it can be changed without making the description inaccurate, put curly braces around it.  Assuming the former:
```suggestion
`go test -v -bench . -benchtime {{50s}}`
```